pkg/controllers/terminating/v1alpha1/suite_test.go

```diff
@@ -75,21 +75,24 @@ var _ = Describe("Reallocation", func() {
 	})
 
 	Context("Reconciliation", func() {
-		It("should terminate nodes marked terminable", func() {
+		It("should terminate nodes marked drained", func() {
 			node := test.NodeWith(test.NodeOptions{
 				Labels: map[string]string{
 					v1alpha1.ProvisionerNameLabelKey:      provisioner.Name,
 					v1alpha1.ProvisionerNamespaceLabelKey: provisioner.Namespace,
-					v1alpha1.ProvisionerPhaseLabel:        v1alpha1.ProvisionerTerminablePhase,
 				},
 				Annotations: map[string]string{
 					v1alpha1.ProvisionerTTLKey: time.Now().Add(time.Duration(-100) * time.Second).Format(time.RFC3339),
 				},
 			})
 			ExpectCreatedWithStatus(env.Client, node)
 
-			ExpectCreated(env.Client, provisioner)
+			persisted := node.DeepCopy()
+			node.Labels[v1alpha1.ProvisionerPhaseLabel] = v1alpha1.ProvisionerDrainedPhase
+			Eventually(Expect(env.Client.Patch(ctx, node, client.MergeFrom(persisted))).To(Succeed()))
```

Comment:
Why is this an eventually?

Reply:
I wanted to do Eventually here to make sure we don't time out if something goes wrong with the environment.

Reply:
By that logic we should wrap the entire world in eventually until nothing ever happens :D

Reply:
Very fair. Thanks.